sys-libs/libxcrypt/libxcrypt-4.4.30-r1.ebuild

```diff
@@ -1,22 +1,16 @@
 # Copyright 2004-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 PYTHON_COMPAT=( python3_{8..11} )
-# NEED_BOOTSTRAP is for developers to quickly generate a tarball
-# for publishing to the tree.
-NEED_BOOTSTRAP="no"
-inherit eapi8-dosym multibuild multilib python-any-r1 flag-o-matic toolchain-funcs multilib-minimal
+VERIFY_SIG_OPENPGP_KEY_PATH="${BROOT}"/usr/share/openpgp-keys/libxcrypt.asc
```

Comment:
I don't get why people put it on top, as if it was a pre-inherit var.

Reply:
I tend to do it for all eclass vars unless there's a reason not to.